test/suite/helper.go

```diff
@@ -115,6 +115,20 @@ func FloatMapMessage(topic, messageType, service string, value map[string]float6
 	}
 }
 
+func IntMapMessage(topic, messageType, service string, value map[string]int64) *fimpgo.Message {
```

Comment:
If you're adding a new function here, please add a correspondent method inside `MessageBuilder` as well to keep the things consistent.